BitwardenShared/UI/Vault/VaultItem/AddEditItem/AddEditItemProcessor.swift

```diff
@@ -680,6 +681,14 @@ final class AddEditItemProcessor: StateProcessor<// swiftlint:disable:this type_
         }
     }
 
+    /// Load the restrict item deletion flag from the config service to state.
+    ///
+    func loadRestrictItemDeletionFlag() async {
+        state.restrictItemDeletionFlagEnabled = await services.configService.getFeatureFlag(
+            FeatureFlag.restrictItemDeletion
```

Comment:
⛏️ I think there's no need to indicate the enum, it should be implicit.
```suggestion
            .restrictItemDeletion
```